geopandas/explore.py

```diff
@@ -204,6 +204,13 @@ def my_colormap(value):  # scalar value defined in 'column'
     highlight_kwds : dict (default {})
         Style to be passed to folium highlight_function. Uses the same keywords
         as ``style_kwds``. When empty, defaults to ``{"fillOpacity": 0.75}``.
+    missing_kwds : dict (default {})
+        Additional style for missing values (NaN):
```

Comment:
```suggestion
        Additional style for missing values:
```

Not only NaN but also None, depending on the dtype.